index.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Awsome Books</title>
+  <link rel="stylesheet" href="index.css">
+</head>
+
+<body>
+  <header>
+    <h2>Awesome Books</h2>
+    <ul>
+      <li class="nav-link">List</li>
+      <li class="nav-link">Add new</li>
+      <li class="nav-link">Contact</li>
+    </ul>
+  </header>
+
+  <div class="date-box">
+    <p class="date"></p>
+  </div>
+
+  <div class="list">
+    <h1 class="title">Awsome Books</h1>
+    <div id="books-container" class="books-container"></div>
+  </div>
+
+
+  <div class="add hide">
+    <hr>
+    <h1 class="title">Add a new Book</h1>
+    <form class="add-form" action="#">
+      <input id="title" type="text" name="title" placeholder="Title">
+      <input id="author" type="text" name="author" placeholder="Author">
+      <button type="button" class="add-btn">Add</button>
+    </form>
```

Comment:
- [ ] [optional] I would suggest resetting the form fields after adding a book to the collection. It will improve the user experience.